src/views/RealityModule/sections/proposal/ProposalSection.tsx

```diff
@@ -165,21 +226,41 @@ export const ProposalSection: React.FC<SectionProps> = ({
                   label="Custom"
                 />
               </RadioGroup>
-            </Grid>
+            </Grid> */}
             <Grid item>
               <ZodiacTextField
                 value={ensName}
-                onChange={({ target }) => setEnsName(target.value)} // TODO: validation
-                label="Enter your snapshot ENS domain."
-                placeholder="ex: gnosis.eth"
-                borderStyle="double"
+                onChange={({ target }) => setEnsName(target.value)}
+                label='Enter your snapshot ENS domain.'
```

Comment:
```suggestion
                label='Enter the Snapshot ENS name.'
```
It seems like `name` is mostly used instead of `domain` (https://docs.ens.domains/frequently-asked-questions).
Also, it should not be "your" name but the Safe´s name.